lib/audited/audit.rb

```diff
@@ -72,16 +72,15 @@ def revision
     # Returns a hash of the changed attributes with the new values
     def new_attributes
       (audited_changes || {}).inject({}.with_indifferent_access) do |attrs, (attr, values)|
-        attrs[attr] = values.is_a?(Array) ? values.last : values
+        attrs[attr] = self.action == 'update' ? values.last : values
         attrs
       end
     end
 
     # Returns a hash of the changed attributes with the old values
     def old_attributes
       (audited_changes || {}).inject({}.with_indifferent_access) do |attrs, (attr, values)|
-        attrs[attr] = Array(values).first
-
+        attrs[attr] = self.action == 'update' ? Array(values).first : values
```

Comment:
Not crucial, but now we don't need to wrap the values as an array since when the action is an `update` the changes are saved as an array.

Reply:
Sure